aws/internal/service/sagemaker/finder/finder.go

```diff
@@ -0,0 +1,25 @@
+package finder
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/sagemaker"
+)
+
+// FincCodeRepositoryByName returns the code repository corresponding to the specified name.
+// Returns nil if no code repository is found.
+func FincCodeRepositoryByName(conn *sagemaker.SageMaker, name string) (*sagemaker.DescribeCodeRepositoryOutput, error) {
```

Comment:
```suggestion
// CodeRepositoryByName returns the code repository corresponding to the specified name.
// Returns nil if no code repository is found.
func CodeRepositoryByName(conn *sagemaker.SageMaker, name string) (*sagemaker.DescribeCodeRepositoryOutput, error) {
```

No need for the `Find` prefix as the package is named `finder`.